scrunch/datasets.py

```diff
@@ -262,28 +295,96 @@ def __str__(self):
         return self.email
 
 
+class Members:
+    """
+    Class for handling collections of members. Adding users,
+    removing users, listing users on a resource...
+    """
+
+    def __init__(self, resource):
+        self.resource = resource
+
+    def list(self):
+        """
+        :return: A list of members of the Entity
+        """
+        members = []
+        for member in self.resource.members.index.values():
+            # members can be users or teams
+            user = member.get('email')
```

Comment:
On which case there is not an email available?

Reply:
if the member it's a Team